.gitignore

```diff
@@ -0,0 +1,19 @@
+# Ignore vi swapfiles
+*.swp
+.*.swp
+
+# Ignore Python cache directories
+__pycache__/
+
+# Ignore project coverage
+.coverage
+
+# Ignore tox working directory
+.tox/
+
+# Ignore "egg info" directories
+*.egg-info
+
+# Ignore the 'env' directory, may be used by developers for a local test
+# environment within the project workspace.
+env/
```

Comment:
This is another one that I might suggest leaving out for the same reason as the vi swap files: it's specific to each individual developer's workflow. Like, personally I use `venv` as the name when creating a local environment with `python -m venv`, or things like `dev-py36` and `dev-py310` when creating them with tox, and I'm sure anyone else who might contribute has their own naming convention for this stuff. Of course it wouldn't be hard for me to use `env/`, but I think it would be equally not hard to leave it up to each individual developer to pick their own names and add those to their local Git ignore rules, and I think that's a more scalable approach.

If anyone does commit files that shouldn't be committed, we should catch it in code review, or if there's a real concern about particular filenames we could always add a pre-commit hook to catch it as well.